All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/src/web_scraping_and_data_wrangling/fos_web_scrap.py

```diff
@@ -0,0 +1,335 @@
+import pandas as pd
+from nltk.corpus import stopwords
+import bs4
+import requests
+from io import BytesIO
+import PyPDF2
+from nltk.tokenize import word_tokenize
+from nltk.tokenize import RegexpTokenizer
+import re
+import string
+import datetime
+import gensim
+from nltk.stem import WordNetLemmatizer, SnowballStemmer
```

Comment:
should be in a requirements file, but also imported in a nice order, e.g. https://pypi.org/project/isort/

Reply:
its done